truffle.js

```diff
@@ -1,41 +0,0 @@
-const HDWalletProvider = require("@truffle/hdwallet-provider");
-require('dotenv').config();
-
-module.exports = {
-  networks: {
-    local: {
-      host: 'localhost',
-      port: 9545,
-      network_id: '*',
-      gas: 4712388
-    },
-    ropsten: {
-      provider: function() {
-        return new HDWalletProvider([process.env.ROPSTEN_KEY], "https://ropsten.infura.io/v3/58a380d3ecd545b2b5b3dad5d2b18bf0");
-      },
-      // host: 'localhost',
-      // port: 8545,
-      network_id: '3',
-      from: '0xa303ddc620aa7d1390baccc8a495508b183fab59'
-      // gas: 5000000
-    }
-  },
-  mocha: {
-    reporter: 'eth-gas-reporter',
-    reporterOptions: {
-      currency: 'USD',
-      gasPrice: 1
-    }
-  },
-  compilers: {
-    solc: {
-      version: "0.7.4",
-    }
-  },
-  plugins: [
-    'truffle-plugin-verify'
```

Comment:
Does hardhat handle etherscan verification automatically? Looks like hardhat has its own `verify` task which you may want to put into package.json https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html#tasks

Reply:
I'll fix this later for Hardhat once we're ready to deploy.